string topic_name
string msg_type #optional, will be PointCloud
---
sensor_msgs/PointCloud cloud
